services/company.service.ts

```diff
@@ -3,36 +3,15 @@ import { ICompany } from "../models/interface/company.model.interface";
 
 export class CompanyService {
 
-    public async createCompany(company: ICompany) {
-        try {
-
-            const companyResponse = await new Company(company);
-            return companyResponse.save();
-
-        } catch(error) {
-            console.error(`CompanyService createCompany error: ${error}`);
-            throw new Error();
-        }
+    public async createCompany(company: ICompany): Promise<ICompany> {

```

Comment:
there should be some validations before creating new company i.e company can be exist with same name or email etc. Any other validations can also be there

Reply:
removing it will do the company CRUD separately